src/lib/schematics/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.__styleext__

```diff
@@ -0,0 +1,3 @@
+table {
```

Comment:
We could also use a class here, but since we also generate the HTML code and want to keep the markup as concise as possible, I've done it that way. This is also consistent with our public examples of the `MatTable`.

Reply:
I would prefer a css class. I don't like that the examples do this and would like to change that too.

Reply:
Makes sense. I've updated it in that case. The other schematics also seem to use classes.

Reply:
`.full-width-table` is good

Reply:
Should be good now.